### ISSUE TITLE
Question about Multi-Cluster Volcano

### ISSUE THREAD

**Issue by stackedsax** [volcano-sh/community#15]
From a recent discussion at the CNCF Research User Group, I was asked to follow up here with a question about multi-cluster plans for Volcano.

I know that we discussed with some Volcano folks about multi-cluster ideas a few years ago but I think Volcano has stayed focused on scheduling within a single cluster since then.  A couple different groups (including us at G-Research) developed multi-cluster batch-scheduling systems because we had that specific need.  I was curious whether the topic of scheduling across multiple k8s clusters has come back up in the Volcano universe and what the current thoughts are there.

Thanks!

**Comment by k82cn** [comment]
Yes, we'd like to start a new repo for multi-cluster; and we'd like to work with community for that. Meanwhile, the multi-cluster will still focus on batch related workloads.

@stackedsax , let's start a design doc to discus the detial if you're also interesting that :)

**Comment by k82cn** [comment]
@stackedsax I re-opened [volcano-sh/federation](https://github.com/volcano-sh/federation) for us to improve Volcano for multi-cluster scenario :)

**Comment by stackedsax** [comment]
Thanks for doing that, @k82cn.  I'm also interested to see where the conversation with the CNCF Batch System Initiative (BSI) Working Group goes.

**Comment by k82cn** [comment]
> Thanks for doing that, @k82cn. I'm also interested to see where the conversation with the CNCF Batch System Initiative (BSI) Working Group goes.

Yes, CNCF BSI WG is a better place for this discussion :)

**Comment by kevin-wangzefeng** [comment]
@stackedsax [Volcano-global](https://github.com/volcano-sh/volcano-global) is already alive to resolve job and queue manangement over multi-cluster architecture, please take a look and feel free to share your feedback.

**Comment by kevin-wangzefeng** [comment]
Closing this issue for now and we can always open a new one :-)
